Codeinwp/neve
# Relative CSS Units Support for Font Size Customizer Controls

### Summary

Relative unit support was added to all font size components (some of them already had partially in stable version)

**The component list:**
* Customizer -> Button -> Button Text -> Font Size
* Customizer -> Form Fields -> Input Text -> Font Size
* Customizer -> Form Fields -> Form Labels -> Font Size
* Customizer -> Typography -> General -> Body -> Font Size
* Customizer -> Typography -> Headings -> H1/H2 to H6 -> Font Size
* Customizer -> Typography -> Blog -> Blog Archive -> (All Font Size controls in the sections of the panel such as Blog Achive->Post Title Blog Archive Post Excerpt, Single Post -> Post Title etc.)

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions

- Make sure all font size components supports relative CSS font size units (rem, em) also supports px.
- Make sure there are no any font size controls which doesn't support relative units (rem, em)

P.S.: I didn't add a support for legacy controls since we have a plan to remove legacy logic from Neve.

### Time
~ 1h 22m

+ ~7m - resolve reviews

Total: 1h 29m


Closes https://github.com/Codeinwp/neve/issues/3785.
